packages/components/src/components/post-tooltip/post-tooltip.tsx

```diff
@@ -35,49 +116,55 @@ export class PostTooltip {
    */
   @Prop() readonly arrow?: boolean = true;
 
-  constructor() {
-    // Create local versions of event handlers for de-registration
-    // https://stackoverflow.com/questions/33859113/javascript-removeeventlistener-not-working-inside-a-class
-    this.localShowTooltip = e => this.show(e.target as HTMLElement);
-    this.localHideTooltip = this.hide.bind(this);
-  }
-
   connectedCallback() {
     if (!this.host.id) {
       throw new Error(
         /*prettier-ignore*/
         'No id set: <post-tooltip> must have an id, linking it to it\'s target element using the data-tooltip-target attribute.',
       );
     }
+  }
 
-    if (!this.triggers) {
-      throw new Error(
-        `No trigger found for <post-tooltip id="${this.host.id}">, please add the 'data-tooltip-target="${this.host.id}" attribute to the trigger element.`,
-      );
+  /**
+   * Add interest event listeners, but only once, and start
```

Comment:
Is an "interest event listeners" a pattern? I never heard about this naming. In my opinion, handlers could be named perhaps differently as for example `globalInterestHandler` doesn't tell me about events.

Reply:
Naming comes from here https://open-ui.org/components/interest-invokers.explainer/, it's like a collective term for hover/focus/long press events.